src/gitHubIssueFetcher.ts

```diff
@@ -99,12 +100,14 @@ export class GitHubIssueFetcher {
         const pageSize = 100;
         const numberOfReposPerCall = this.config.numberOfReposPerCall;
         const reposForEachCall = this.splitArray(repositoryIdentifiers, numberOfReposPerCall);
+        const daysAgoCreated = this.config.daysAgoCreated != undefined ? this.config.daysAgoCreated : 90;
+        const dateStringToQuery = DateTime.local().minus({ days: daysAgoCreated }).toISODate();
```

Comment:
Answering the questions in the PR header: 

1. Normally I like to keep all my thinking in UTC, but local here seems fine. I don't have a preference. 

2. toISODate (excluding the time) seems best here. (I don't know if GitHub allows a full time string or not.)